woocommerce/woocommerce-gateway-stripe
# Remove provide feedback link from upe settings

## Changes proposed in this Pull Request

Fixes #2098 

- Remove the **Provide feedback** link from the drop-down menu in the **Payment Methods** section of the UPE settings.

## Testing instructions

1. Make sure UPE (new checkout experience) is enabled.
2. Go to **WooCommerce** > **Settings** > **Payments** > **Stripe**.
3. Click the kebab menu to show the drop-down.
4. Notice that there no **Provide feedback link**.

<img width="1315" alt="image" src="https://user-images.githubusercontent.com/13835680/139602712-3004ecdb-0eff-4b3d-b9bc-189789df76ac.png">


| Before | After
|:---:|:---:|
| ![](https://user-images.githubusercontent.com/10233985/138515778-ef02d8dc-c621-46a3-93cd-9d6075168235.png) | <img width="312" alt="image" src="https://user-images.githubusercontent.com/13835680/139602731-0e02f12e-8f47-44e9-9b1b-d79c9bbdf969.png"> |



---
